nat-lab/tests/telio_features.py

```diff
@@ -80,6 +80,7 @@ class Nurse(DataClassJsonMixin):
 class LinkDetection(DataClassJsonMixin):
     rtt_seconds: Optional[int] = None
     no_of_pings: Optional[int] = 0
+    use_for_downgrade: Optional[bool] = False
```

Comment:
There's some testing for these `TelioFeatures`, let's add this field to that test as well

Reply:
Ok, on it